src/chart/main/chart.cpp

```diff
@@ -147,6 +147,40 @@ void Chart::draw(Gfx::ICanvas &canvas)
 			    }
 		    });
 
+		actPlot->getOptions()->subtitle.visit(
```

Comment:
Similar code 3 times, maybe extract this to a function?

Reply:
OK: https://github.com/vizzuhq/vizzu-lib/pull/349/commits/4e005204c67ac3a7a30d222ad0e2c2227aaa3579